tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -732,6 +732,54 @@ def test_optuna_callback(self) -> None:
 
         assert callback_mock.call_count == 10
 
+    def test_tune_best_params_reproducibility(self) -> None:
+        import sklearn.datasets
+        from sklearn.model_selection import train_test_split
+
+        from optuna.samplers import TPESampler
+        import optuna.study as study
```

Comment:
I'm wondering if these lines move to the top of this test file?

Reply:
Done. I moved to top